vsts/pipelines/validation.yml

```diff
@@ -29,8 +29,24 @@ jobs:
     displayName: 'Set variables'
   - template: templates/_buildTemplate.yml
 
-- job: Job_RuntimeImages
-  displayName: Build and Test Runtime Images
+- job: Job_Stretch_RuntimeImages
```

Comment:
This job does not anything different than the later `buster` one. I would suggest to not add it until we make changes to our tests which probably can be decorated with a test trait maybe.

Reply:
so are you suggesting to (for now) just to build the image and push through prod?

Reply:
No, i mean our tests can continue testing the `buster` based images as they are already doing for `stretch` based images. since this particular is not really filtering based on flavor, we are executing all the tests 2 times